libsplinter/src/admin/service/event/store/diesel/mod.rs

```diff
@@ -14,6 +14,12 @@
 
 //! Database backend support for the `AdminServiceEventStore`, powered by
 //! [`Diesel`](https://crates.io/crates/diesel).
+//!
```

Comment:
Several items listed in the commit message are actually in the last commit.

Reply:
Updated to add the traits / impls mentioned in the commit message